zalando-incubator/kubernetes-on-aws
# Update to Kubernetes 1.11

Let's see what the e2es have to say about this.

Changes:
* Update Hyperkube to 1.11.x
* Update CoreOS to the latest stable (as of July 4)
* Remove PodPriority feature flag as it's enabled by default
* ~~Switch from KubeProxy's `iptables` mode to `ipvs` mode~~
* ~~Disable docker version change, therefore it uses 18.03.x with the AMI above which is **not** officially supported by Kubernetes 1.11 (let's see where we end up; 1.12 and 17.03.x work fine and are supported)~~
* Remove `PersistentVolumeClaimResize` feature flag ([enabled by default](https://kubernetes.io/blog/2018/07/12/resizing-persistent-volumes-using-kubernetes/?utm_source=cncf&utm_medium=email&utm_campaign=cloudnativemonthly&utm_content=july18))
* Remove `CustomResourceSubresources` feature gate ([enabled by default](https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/#subresources)).
